### PR TITLE
docs: Grafana Dashboard Configuration

### PR DESCRIPTION
**Pull request by Sindweller**
Please answer these questions before submitting a pull request
- Why submit this pull request?
- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues
___
### New feature or improvement
- Describe the details and related test reports.
This pr has a prior pr #670 .
A shortcut of Grafana Dashboard for apisix-ingress-controller:
grafana-1
grafana-2

grafana-3

### CODE REVIEW

**Comment by tao12345666333** [comment]
ping @Sindweller any updates?

**Comment by tao12345666333** [comment]
Can you add a document describing how to monitor and use this dashboard? Thanks 

If you need any help please feel free to ping me.

**Review by tokers — APPROVED** [review]

**Comment by tokers** [comment]
@Sindweller What does the `cluster health check` value mean?

**Comment by Sindweller** [comment]
> Can you add a document describing how to monitor and use this dashboard? Thanks
> 
> If you need any help please feel free to ping me.

working on it now.

**Comment by Sindweller** [comment]
> @Sindweller What does the `cluster health check` value mean?

It increases when [checkClusterHealth](https://github.com/apache/apisix-ingress-controller/blob/f4708675c6304ad019881ad7e0ac7a0affd3e6bd/pkg/ingress/controller.go#L642) is called to record a successful health check for default cluster.

**Comment by tokers** [comment]
> > @Sindweller What does the `cluster health check` value mean?
> 
> It increases when [checkClusterHealth](https://github.com/apache/apisix-ingress-controller/blob/f4708675c6304ad019881ad7e0ac7a0affd3e6bd/pkg/ingress/controller.go#L642) is called to record a successful health check for default cluster.

OK, so what about APISIX Health Check Calls, so that we know the usage of this metrics.

File: docs/en/latest/plugins/prometheus.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+-->
+
+This guide shows how to monitor Apache APISIX Ingress Controller using Prometheus and Grafana.
+
+## Enable Prometheus
+
+You can find example CRD files in `apisix-ingress-controller/samples/deploy/crd/v1`.  
+The CRD file `ApisixClusterConfig` is required to use to enable prometheus in global configurations.  
+In `ApisixClusterConfig`, `monitoring.prometheus` field is used to describe Prometheus settings.
+```yaml
```
Comment (tao12345666333) [inline docs/en/latest/plugins/prometheus.md:31]: This file is a definition file for custom resources. In order to be able to configure correctly, we should create a custom resource configuration file, like this:

```yaml
apiVersion: apisix.apache.org/v2beta3
kind: ApisixClusterConfig
metadata:
  name: default
spec:
  monitoring:
    prometheus:
      enable: true
```

**Review by tao12345666333 — CHANGES_REQUESTED** [review]

File: docs/en/latest/plugins/prometheus.md (hunk 1 above)
Comment (Sindweller) [reply docs/en/latest/plugins/prometheus.md:29]: please see if I understand the usage of crd file.

Comment (tao12345666333) [reply docs/en/latest/plugins/prometheus.md:29]: looks good.

**Comment by tao12345666333** [comment]
@Sindweller Thanks! It looks great to me!

Please fix the markdown lint error. Then we can merge this one.

**Review by tao12345666333 — APPROVED** [review]

**Comment by tao12345666333** [comment]
@Sindweller LGTM. Thanks!

**Review by gxthrj — APPROVED** [review]

**Pull request merged**